src/usr/lib/rsetup/cli/wi-fi.sh

```diff
@@ -3,14 +3,32 @@
 ALLOWED_RCONFIG_FUNC+=("connect_wi-fi")
 
 connect_wi-fi() {
+    local i ssid="$1" password="$2" total_retry=10
+    local command=("nmcli device wifi connect $ssid")
     case $# in
-        1)
+        1|2)
             nmcli radio wifi on
-            nmcli dev wifi connect "$1"
-            ;;
-        2)
-            nmcli radio wifi on
-            nmcli dev wifi connect "$1" password "$2"
+
+            if (( $# == 2 ))
+            then
+                command+=(password "$password")
+            fi
+
+            for ((i = 0; i < total_retry; i++))
+            do
+                # shellcheck disable=SC2068
+                if ${command[@]}
+                then
+                    echo "Wi-Fi successfully connected to '$ssid'."
+                    return
+                else
+                    echo "Failed to connect to '$ssid'. Retry $((i + 1)) of '$total_retry'." >&2
```

Comment:
```suggestion
                    echo "Failed to connect to '$ssid'. Retry $((i + 1)) of $total_retry." >&2
```
total_retry是我们自己定义的变量，并且知道是个数字，所以没必要用引号包括